src/test/resources/same/literals/Literals.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func main() {
+    param1 := 1
+    param2 := "stringParam2"
+    param3 := 0xFFFFFFFFFFFF
+    var param4 uint64 = 0xff1a618b7f65ea12
+    var param5 uint64 = 0xc4ceb9fe1a85ec53
+    param6 := 0757
+    param7 := 0123546263753256452432
+    param8 := 123.E+2
+    param9 := 1e-1
+    param10 := -1e-1
+    param11 := 1.e+0i  // complex (imaginary) number
+
+    fmt.Println(param1,param2,param3,param4,param5)
```

Comment:
You can remove the `fmt.Prtinln`s

Reply:
Actually this is just for making the file a well-formed Go file as Go prohibits variables which aren't used.